.github/workflows/prod_deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy ECS
+
+on:
+  workflow_dispatch:
+    inputs:
+      stack:
+        type: choice
+        descriptions: The layer of the stack to deploy
+        options:
+          - api
```

Comment:
Should we comment this out as well as a `todo`, until the action exists?

Reply:
Done in 4f6db59.